src/com/sap/piper/PiperGoUtils.groovy

```diff
@@ -67,15 +67,11 @@ class PiperGoUtils implements Serializable {
     }
 
     private boolean downloadGoBinary(url) {
-
         try {
-            def httpStatus = steps.sh(returnStdout: true, script: "curl --insecure --silent --location --write-out '%{http_code}' --output ./piper '${url}'")
-
-            if (httpStatus == '200') {
-                steps.sh(script: 'chmod +x ./piper')
-                return true
-            }
-        } catch(err) {
+            steps.httpRequest ignoreSslErrors: true, outputFile: 'piper', url: url, validResponseCodes: '200'
```

Comment:
do we follow redirects here?

GitHub can respond with a redirect which needs to be followed.
This was do far ensured by `--location`

Reply:
> do we follow redirects here?

yes, it follows redirects automagically. I checked the details wrt this earlier when changing something in the tms area.